### PR TITLE
Models created, migration done and repository pattern implemented

### PR DESCRIPTION
**Pull request by Ayomipo18**

### CODE REVIEW

**Review by anabiz — CHANGES_REQUESTED** [review]

File: CompanyEmployees/Migrations/20220616172911_DatabaseCreation.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
```
[the remark this answers is not in the document]
Comment (Ayomipo18) [reply CompanyEmployees/Migrations/20220616172911_DatabaseCreation.cs]: Done

File: CompanyEmployees/Migrations/20220616215216_InitialData.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
```
[the remark this answers is not in the document]
Comment (Ayomipo18) [reply CompanyEmployees/Migrations/20220616215216_InitialData.cs]: Done

File: CompanyEmployees/Migrations/20220616215216_InitialData.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+#nullable disable
+
+namespace CompanyEmployees.Migrations
```
[the remark this answers is not in the document]
Comment (Ayomipo18) [reply CompanyEmployees/Migrations/20220616215216_InitialData.cs]: Done

File: CompanyEmployees/Migrations/RepositoryContextModelSnapshot.cs [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using Repository;
+
+#nullable disable
+
+namespace CompanyEmployees.Migrations
```
[the remark this answers is not in the document]
Comment (Ayomipo18) [reply CompanyEmployees/Migrations/RepositoryContextModelSnapshot.cs]: Done

File: CompanyEmployees/Migrations/20220616215216_InitialData.cs [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+{
+    public partial class InitialData : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```
[the remark this answers is not in the document]
Comment (Ayomipo18) [reply CompanyEmployees/Migrations/20220616215216_InitialData.cs]: Done

File: CompanyEmployees/appsettings.json [hunk 6]
```diff
@@ -5,5 +5,8 @@
       "Microsoft.AspNetCore": "Warning"
     }
```
[the remark this answers is not in the document]
Comment (Ayomipo18) [reply CompanyEmployees/appsettings.json]: Done

**Pull request merged**